src/renderer/src/pages/game-details/modals/game-options-modal.tsx

```diff
@@ -226,6 +244,33 @@ export function GameOptionsModal({
             </div>
           )}
 
+          {shouldShowLaunchOptionsConfiguration && (
+            <div className={styles.gameOptionHeader}>
+              <h2>{t("launch_options")}</h2>
+              <h4 className={styles.gameOptionHeaderDescription}>
+                {t("launch_options_description")}
+              </h4>
+              <TextField
+                value={launchOptions}
+                theme="dark"
+                placeholder={t("launch_options_placeholder")}
+                onChange={handleChangeLaunchOptions}
+                rightContent={
+                  <>
+                    {game.launchOptions && (
+                      <Button
+                        onClick={handleClearLaunchOptions}
+                        theme="outline"
+                      >
+                        {t("clear")}
+                      </Button>
+                    )}
+                  </>
```

Comment:
Here there's no need to add a react fragment, as there only one element. this is enough:
```suggestion
                  game.launchOptions && (
                    <Button onClick={handleClearLaunchOptions} theme="outline">
                      {t("clear")}
                    </Button>
                  )
```

Reply:
Fragment removed